notebooks/240-efficientdet-object-detection/240-efficientdet-object-detection.ipynb

```diff
@@ -0,0 +1,666 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        img = img.reshape([1,h,w,3])
</pre><p>The image is already in HWC. Just add a new dimension at the beginning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>